### ISSUE TITLE
Setting disableDragAndDropProvider={true} in <FullFileBrower/> error

### ISSUE THREAD

**Issue by simonbarker** [TimboKZ/Chonky#46]
```
<FullFileBrowser 
              files={files} 
              folderChain={folderChain} 
              onFileAction={handleFileAction}
              defaultFileViewActionId="enable_list_view"
          />
```

causes the app to not load with the following error:

Expected drag drop context
invariant
src/index.ts:28
  25 |   );
  26 | } else {
  27 |   let argIndex = 0;
> 28 |   error = new Error(
     | ^  29 |     format.replace(/%s/g, function() {
  30 |       return args[argIndex++];
  31 |     })
View compiled

let me know what other info you need. Tried this on 2.0.0, 2.0.1 and 2.0.3

**Comment by TheAksh** [comment]
disableDragAndDrop={true}

Gives an error as well.

**Comment by TimboKZ** [comment]
I just released [v2.0.5](https://github.com/TimboKZ/Chonky/releases/tag/chonky%402.0.5) that should fix this issue. Can you try installing it?
```bash
npm install chonky-icon-fontawesome@latest chonky@latest
```

**Comment by simonbarker** [comment]
@TimboKZ that's done the trick thanks 😀 good job we chatted on Discord as I've just seen that the snippet I pasted in the initial comment doesn't actually have  `disableDragAndDrop={true}` in the props! 🤦‍♂️
